macos-mojave_5c6d3bfa2c7d3a66e32eaf2e.md

```diff
@@ -97,6 +97,10 @@ Installed versions:
 - 2.3.7 (system)
 - 2.5.1
 
+## Flutter
+
+- v1.2.1-stable (macOS)
```

Comment:
@CommanderK5 is `macOS` mentioned here because there's an Android edition?

Reply:
@markoa there are version for Windows and Linux, not sure why it ended up here. I can remove it if you don't mind.

Reply:
It just made me wonder what it means, may happen to others too, so 👍 for removing it.